scripts/generate_badges.d

```diff
@@ -1,13 +1,20 @@
-#!/usr/bin/env -S docker run --rm -it -v .:/src dlanguage/dmd dmd -run generate_badges.d
+#!/usr/bin/env -S docker run --rm -v ${PWD}/scripts:/scripts -v ${PWD}/.github/workflows:/.github/workflows dlanguage/dmd dmd -run /scripts/generate_badges.d
+
+// To run this script:
+// cd /path/to/cpu_features
+// chmod +x scripts/generate_badges.d
```

Comment:
this file is now executable (cf my previous PR) not sure you need this comment anymore